lib/guardian/post_guardian.rb

```diff
@@ -78,7 +78,7 @@ def post_can_act?(post, action_key, opts: {}, can_see_post: nil)
         (
           is_flag && not(already_did_flagging) &&
             (
-              @user.has_trust_level?(TrustLevel[SiteSetting.min_trust_to_flag_posts]) ||
```

Comment:
@lis2 `User#has_trust_level?` does a few other checks, e.g. if the user is admin or moderator. Shall I add those groups as well, in the migration? 🤔

Reply:
I think it is fine, especially that default is TL1 so original requirement is quite low